src/main/java/cl/transbank/pos/POS.java

```diff
@@ -79,6 +80,7 @@ public static POS getInstance() throws TransbankLinkException {
             }
             try {
                 System.load(nativeTransbankWrapper);
+                // TransbankWrap.list_ports();
```

Comment:
We don't comment code, you can delete it, and if you need it back, you can use git to recoverit.